toolkits/x/arcade_x/tools/tweets.py

```diff
@@ -64,6 +65,9 @@ async def delete_tweet_by_id(
 async def search_recent_tweets_by_username(
     context: ToolContext,
     username: Annotated[str, "The username of the X (Twitter) user to look up"],
+    return_attachments_metadata: Annotated[
```

Comment:
nit: since the payload of this is minimal we could forego this being a parameter and just always return it. I'll leave it up to you though!